src/core/src/main/java/org/apache/jmeter/util/JsseSSLManager.java

```diff
@@ -236,7 +237,8 @@ private SSLContext createContext() throws GeneralSecurityException {
 
         // Wrap the defaults in our custom trust manager
         TrustManager[] trustmanagers = tmfactory.getTrustManagers();
-        for (int i = 0; i < trustmanagers.length; i++) {
+        int trustManagersCount = trustmanagers.length;
+        for (int i = 0; i < trustManagersCount; i++) {
```

Comment:
WDYT of for-each loop?

Reply:
Same here, are you agree with new version with index?

```
int indexTrustManager = 0;
        for (TrustManager trustManager : trustmanagers) {
            if (trustManager instanceof X509TrustManager) {
                trustmanagers[indexTrustManager] = new CustomX509TrustManager(
                        (X509TrustManager) trustManager);
            }
            indexTrustManager++;
        }
```

Reply:
old code reverted @vlsi